pkgs/top-level/python-packages.nix

```diff
@@ -5180,6 +5180,8 @@ self: super: with self; {
 
   indexed_gzip = callPackage ../development/python-modules/indexed_gzip { };
 
+  indexed_zstd = callPackage ../development/python-modules/indexed_zstd { };
```

Comment:
Woops. I guess do it the proper way: https://github.com/NixOS/nixpkgs/pull/121412#discussion_r624540323

Reply:
Ah thanks, I didn't understand the comment on my first reading because of the mention of Docker. My association was that it would build the package in a Docker container, but by involuntarily triggering the zstd.h error and seeing your pkgs.zstd.dev solution, it makes sense now.